src/components/Item/index.js

```diff
@@ -1,9 +1,22 @@
 import "./item.css";
 
 function Item(props) {
+
+  const numbers = [1, 2, 3, 4, 5];
+  const stars = numbers.map((number) =>
+  <div  className={((number>0)&&(number<=props.stars)) ? 'list__content--active' : 'list__content--off'}><i class="fas fa-star"></i></div>  
+  );
+
   return (
-    <li key={props.id}>
-      {props.name} {props.city}
+    <li key={props.id} className="list">
```

Comment:
Please change key to something very unique like 

```
`hotel-element-main-list-{props.id}`
```

Reply:
Fixed - renaming key